janus_core/helpers/utils.py

```diff
@@ -166,54 +149,6 @@ def _build_filename(
         return built_filename
 
 
-def spacegroup(
-    struct: Atoms, sym_tolerance: float = 0.001, angle_tolerance: float = -1.0
-) -> str:
-    """
-    Determine the spacegroup for a structure.
-
-    Parameters
-    ----------
-    struct : Atoms
-        Structure as an ase Atoms object.
-    sym_tolerance : float
-        Atom displacement tolerance for spglib symmetry determination, in Å.
-        Default is 0.001.
-    angle_tolerance : float
-        Angle precision for spglib symmetry determination, in degrees. Default is -1.0,
-        which means an internally optimized routine is used to judge symmetry.
-
-    Returns
-    -------
-    str
-        Spacegroup name.
-    """
-    return get_spacegroup(
-        cell=(
-            struct.get_cell(),
-            struct.get_scaled_positions(),
-            struct.get_atomic_numbers(),
-        ),
-        symprec=sym_tolerance,
-        angle_tolerance=angle_tolerance,
-    )
-
-
-def snap_symmetry(struct: Atoms, sym_tolerance: float = 0.001) -> None:
-    """
-    Symmetrize structure's cell vectors and atomic positions.
-
-    Parameters
-    ----------
-    struct : Atoms
-        Structure as an ase Atoms object.
-    sym_tolerance : float
-        Atom displacement tolerance for spglib symmetry determination, in Å.
-        Default is 0.001.
-    """
-    refine_symmetry(struct, symprec=sym_tolerance, verbose=False)
-
-
 def none_to_dict(dictionaries: Sequence[Optional[dict]]) -> Generator[dict, None, None]:
     """
     Ensure dictionaries that may be None are dictionaires.
```

Comment:
minor typo

```suggestion
    Ensure dictionaries that may be None are dictionaries.
```

and this below but I don't seem to be able to make a suggestion on it

```python
dictionaries : Sequence[dict]
        Sequence of dictionaries that be be None. # could be None?
```

Reply:
Thanks!